dodam-teacher-ios/iosApp/ContentView.swift

```diff
@@ -13,7 +13,8 @@ struct ComposeView: UIViewControllerRepresentable {
 struct ContentView: View {
     var body: some View {
         ComposeView()
-                .ignoresSafeArea(.keyboard) // Compose has own keyboard handler
+            .ignoresSafeArea(.all)
```

Comment:
why are you ignore all of safe area? I think it would be better don't ignore top safe area